srv/modules/runners/changed.py

```diff
@@ -36,7 +37,7 @@ def __init__(self, **kwargs):
         self.conf_dir = kwargs.get('conf_dir', '/srv/salt/ceph/configuration/files/ceph.conf.d/')
         self.conf_filename = kwargs.get('conf_filename', self._role_name)
         self.conf_extension = kwargs.get('conf_extension', '.conf')
-        self._conf_files = [self.conf_dir + self.conf_filename + self.conf_extension]
+        self._conf_files = glob.glob(self.conf_dir + self.conf_filename + self.conf_extension)
```

Comment:
My assumption:

in the [test_changed.py](https://github.com/SUSE/DeepSea/blob/master/tests/unit/runners/test_changed.py#L34) we create a FakeFs File. Since your change now uses `glob.glob` this file can't be found.

```
(Pdb) self.role.conf_files
[]
(Pdb) self.role.name
'rgw'
```
If we decide to use `glob.glob` we may even drop the FakeFs alltogether, which I'd be in favor of.